### ISSUE TITLE
训练一个分类器 | 报错解决方案

### ISSUE THREAD

**Issue by KaiJiaBrother** [apachecn/pytorch-doc-zh#483]
在[训练一个分类器](https://pytorch.apachecn.org/docs/0.3/blitz_cifar10_tutorial.html)这里，4. 训练网络第21行代码会报错，建议改为：running_loss += loss.item()

**Comment by LovelyBuggies** [comment]
@KaiJiaBrother 现在已经出新版本的中文文档了，你可以看一下latest version

**Comment by LovelyBuggies** [comment]
PyTorch 0.3 官方就是这样写的，现在的1.4版本已经更新为 `item()`。
